cpg-concepts/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/concepts/auth/Authentication.kt

```diff
@@ -68,7 +68,7 @@ open class JwtAuth(underlyingNode: Node? = null, val jwt: Node, val payload: Nod
 
 /** Abstract base class for authentication operations. */
 abstract class AuthenticationOperation(underlyingNode: Node? = null, concept: Authentication) :
-    Operation(underlyingNode, concept)
+    Operation(concept, underlyingNode)
```

Comment:
The order of parameters and arguments does not match due to changes in `Operation`. Please refactor `AuthenticationOperation` to match order.